tox.ini

```diff
@@ -1,12 +1,19 @@
-# content of: tox.ini , put in same dir as setup.py
 [tox]
 envlist = py27,py35
 requires =
     pip >= 18.1
 
 [testenv]
 deps =
-    -r requirements.txt
     -r dev-requirements.txt
+
+# envsitepackagesdir lets pytest-cov find the source where tox actually
```

Comment:
Does this tell pytest to actually use the installed and not the code in the root?

Reply:
Yep. The paths in the coverage report look like this:
```
.tox/py35/lib/python3.5/site-packages/cdisutils/parsers.py
```

I think it is possible to supply package names to the coverage tool itself, but I don't think pytest-cov supports that.